src/pages/ConnectWallet/ConnectWallet.tsx

```diff
@@ -178,83 +157,62 @@ export const ConnectWallet = () => {
-                      </Tooltip>
-                    </HStack>
-                  </Stack>
-                  <Button
-                    width='full'
-                    size='lg-multiline'
-                    onClick={handleMetaMaskConnect}
-                    leftIcon={metamaskIcon}
-                  >
-                    {translate('walletProvider.metaMaskSnap.connectMetaMask')}
-                  </Button>
-                </Stack>
-                <Flex alignItems='center' justifyContent='center' width='full' gap={4}>
-                  <Divider flex={1} borderColor='border.bold' opacity='1' />
-                  <Text
-                    color='text.subtle'
-                    fontWeight='medium'
-                    textAlign='center'
-                    translation='common.or'
-                  />
-                  <Divider flex={1} borderColor='border.bold' opacity='1' />
-                </Flex>
-              </>
-            ) : (
-              <Stack spacing={6}>
-                <Heading as='h3' fontSize='2xl'>
-                  {translate('connectWalletPage.welcomeBack')}
+            <Stack spacing={6}>
+              <Stack spacing={4} textAlign='center' alignItems='center' fontWeight='bold'>
+                <RawText color='blue.300' fontSize='m' textTransform='uppercase'>
+                  {translate('connectWalletPage.secondaryTitle')}
+                </RawText>
+                <Heading as='h1' fontSize='4xl'>
+                  {translate('connectWalletPage.primaryTitle')}
                 </Heading>
-                <RawText fontSize='lg' color='text.subtle'>
-                  {translate('connectWalletPage.welcomeBody')}
+                <RawText color='text.subtle' fontSize='m'>
+                  {translate('connectWalletPage.primaryDescription')}
                 </RawText>
+                <Button
+                  size='lg-multiline'
+                  zIndex={1}
+                  colorScheme='blue'
+                  width='fit-content'
+                  onClick={handleConnectClick}
+                  data-test='connect-wallet-button'
+                  my={4}
+                >
+                  <Text translation='connectWalletPage.cta' />
+                </Button>
+                {isSnapEnabled && (
+                  <>
+                    <Flex alignItems='center' justifyContent='center' width='full' gap={4}>
+                      <Divider flex={1} borderColor='border.bold' opacity='1' />
+                      <Text
+                        color='text.subtle'
+                        fontWeight='medium'
+                        textAlign='center'
+                        translation='common.or'
+                        textTransform='uppercase'
+                      />
+                      <Divider flex={1} borderColor='border.bold' opacity='1' />
+                    </Flex>
+                    <Box bg='background.button.secondary.base' p={6} borderRadius='lg' mt={4}>
```

Comment:
```suggestion
                    <Box bg='whiteAlpha.50' p={8} borderRadius='lg' mt={4}>
```

To be equal to mockups

Reply:
Addressed in https://github.com/shapeshift/web/pull/7793/commits/09dc620fb5154c0de092a12416e1bcdcbf403e64